src/common/filesystem.c

```diff
@@ -715,19 +715,35 @@ FS_LoadPAK(const char *packPath)
 	if (LittleLong(header.ident) != IDPAKHEADER)
 	{
 		fclose(handle);
-		Com_Error(ERR_FATAL, "FS_LoadPAK: '%s' is not a pack file", packPath);
+		Com_Error(ERR_FATAL, "%s: '%s' is not a pack file", __func__, packPath);
+		return NULL;
```

Comment:
why return NULL; ? Com_Error() exits the game

Reply:
It was for cppcheck. I have removed it as not required.